src/index.js

```diff
@@ -670,10 +681,35 @@ async function jsPDFInvoiceTemplate(props) {
         indiaIRP.irn
       );
       currentHeight += pdfConfig.subLineHeight;
+    } else {
+      doc.setFont(undefined, FONT_TYPE_BOLD);
+      currentHeight += pdfConfig.subLineHeight;
+    }
+
+    if (
+      indiaIRP.ackDate &&
+      indiaIRP.ackNumber &&
+      indiaIRP.isAckDetailsInInvPDF
```

Comment:
should we keep the flag outside of pdf component? 
Like when we call the pdf, we check if to pass ackDate and ackNumber

Reply:
so this flag is already outside inv pdf - if we are not passing it from code then it won't show it at all